bindings/nodejs/lib/client/Client.ts

```diff
@@ -273,9 +274,7 @@ export class Client {
      */
     async signatureUnlock(
         secretManager: SecretManagerType,
-        // Uses `Array<number>` instead of `Uint8Array` because the latter serializes
-        // as an object rather than an array, which results in errors with serde.
-        transactionEssenceHash: Array<number>,
+        transactionEssenceHash: HexEncodedString,
```

Comment:
Should we also do this to the current nodejs bindings?

Reply:
Client nodejs binding got removed and in the wallet we don't have it

Reply:
Ah right, my bad